### PR TITLE
LPD-15580 Adapt tests

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/LocalConfigContext.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	return useCallback(
+		(itemId: string, itemConfig: ItemConfig) => {
+			const currentItemConfig = config[itemId] || {};
+
+			const nextConfig = setIn(config, [itemId], {
+				...currentItemConfig,
+				...itemConfig,
+			});
+
+			setConfig(nextConfig);
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/LocalConfigContext.tsx:64]: I believe we don't need to pass the preview config here, we can access it in the setConfig function like

`setConfig(prevConfig => {...});`

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/contexts/LocalConfigContext.tsx:64]: Makes sense! Done ✅😄

**Comment by victorg1991** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:test:report:2533325262

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:report:2533588869

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**